### PR TITLE
luci-app-https-dns-proxy-js: Add he.net as provider

### PR DESCRIPTION
**Pull request by systemcrash**

### CODE REVIEW

**Comment by stangri** [comment]
Hey Paul,

this is just a placeholder for when I have the time to convert the lua app to javascript, it is not being built or used in any way right now.

If you want to add HE to the list of the providers in the current app, that would be `luci-app-https-dns-proxy` either in this repo or you can send a polished PR directly to github.com/openwrt/luci repo.

File: luci-app-https-dns-proxy-js/root/usr/share/https-dns-proxy/providers/net.he.ordns.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"name": "HurricaneElectric",
+	"label":_("HurricaneElectric"),
```
Comment (stangri) [inline luci-app-https-dns-proxy-js/root/usr/share/https-dns-proxy/providers/net.he.ordns.json]: Please change to: `"label":_("Hurricane Electric"),`

File: luci-app-https-dns-proxy/luasrc/https-dns-proxy/providers/net.he.ordns.lua [hunk 2]
```diff
@@ -0,0 +1,8 @@
+return {
+        name = "HurricaneElectric",
+        label = _("HurricaneElectric"),
```
Comment (stangri) [inline luci-app-https-dns-proxy/luasrc/https-dns-proxy/providers/net.he.ordns.lua]: Please change to `label = _("Hurricane Electric"),`

**Review by stangri — CHANGES_REQUESTED** [review]

**Review by stangri — APPROVED** [review]

**Comment by systemcrash** [comment]
> If you want to add HE to the list of the providers in the current app, that would be `luci-app-https-dns-proxy` either in this repo or you can send a polished PR directly to github.com/openwrt/luci repo.

I'll try and get that squeezed in there...

**Comment by stangri** [comment]
> > If you want to add HE to the list of the providers in the current app, that would be `luci-app-https-dns-proxy` either in this repo or you can send a polished PR directly to github.com/openwrt/luci repo.
> 
> I'll try and get that squeezed in there...

Shouldn't be hard, just squash all the commits into a single one or do 'git commit --amend' if you need to add more changes in the future.

**Pull request merged**